packages/embarkjs/src/blockchain.js

```diff
@@ -172,7 +172,7 @@ Blockchain.doConnect = function(connectionList, opts, doneCb) {
 
     if (connectionString === '$WEB3') {
       connectWeb3(next);
-    } else if (connectionString.indexOf('ws://') >= 0) {
+    } else if (connectionString.indexOf('wss://') >= 0 || connectionString.indexOf('ws://') >= 0) {
```

Comment:
Maybe a regex test would be simpler here?

```suggestion
    } else if (/^wss?:\/\//.test(connectionString)) {
```

Reply:
Agree

Reply:
Updated. Thanks for the suggestion. I knew the regex was better, but I was too lazy so I went the easier way 😆 🙈